library/ajax/weno_pharmacy_search.php

```diff
@@ -0,0 +1,124 @@
+<?php
```

Comment:
will this script need an access control(s) (ie. what type of users will need access to this script?).

Reply:
it is an ajax for a pharmacy lookup when creating/editing a patient's pharmacy. I would include csrf to the script in the next commit